postgres/lib.test.w

```diff
@@ -2,11 +2,10 @@ bring expect;
 bring util;
 bring "./lib.w" as l;
 
-if util.env("WING_TARGET") == "tf-aws" {
-  let db = new l.Database(name: "test", pgVersion: 15);
+let db = new l.Database(name: "test", pgVersion: 15);
 
-  test "run a simple query" {
-    let result = db.query("SELECT 1 as one, 2 as two;");
-    expect.equal(result.at(0), {one: 1, two: 2});
-  }
+test "run a simple query" {
+  util.sleep(1s);
```

Comment:
Why is this needed?

Reply:
Ah I think I was debugging something, it should be possible to remove it

Reply:
Looks like we do need some kind of command-based readiness check.. for now I'm just retrying connecting